config/www/cgi/cgi.py

```
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+import cgi
```

Ruff

`cgi` imported but unused

Remove unused import: `cgi`

(F401)